lib/pesticide/usage.rb

```diff
@@ -4,13 +4,14 @@ class Usage
 
     def initialize(attributes = {})
       @name = attributes.delete(:name)
-      @dose = Measure.new(attributes.delete(:dose)) if attributes[:dose]
+      @dose = (attributes[:dose] && attributes[:dose] =~ /\A-?([\,\.]\d+|\d+([\,\.]\d+)?)\s*[^\s]+\z/) ? Measure.new(attributes.delete(:dose)) : attributes.delete(:dose)
```

Comment:
@burisu Do you want me to create another issue specific to dose measure or to edit the current one to include dose measure alongside working_zone?